README.md

```diff
@@ -98,12 +98,12 @@ kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
 kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
 ```
 
-Once the installation is completed, you can test the deployment of an image that is signed using Notary V2 solution.
+Once the installation is completed, you can test the deployment of an image that is signed using Notation solution.
 
 - This will successfully create the pod `demo`
 
 ```bash
-kubectl run demo --image=ghcr.io/deislabs/ratify/notary-image:signed
+kubectl run demo --image=ghcr.io/deislabs/ratify/notation-image:signed
```

Comment:
I don't think the image in ghcr is published at this new repository. cc: @susanshi